haproxy/default.toml

```diff
@@ -1,6 +1,7 @@
 maxconn = 32
 mode = "http"
 bind = "*:80"
+httpchk = "GET /favicon.ico"
```

Comment:
I agree! Will update when I get some spare cycles - just trying to share out the progress we make at $job :)

> On 12 Aug 2016, at 14:23, Mike Fiedler <notifications@github.com> wrote:
> 
> In haproxy/default.toml:
> 
> > @@ -1,6 +1,7 @@
> >  maxconn = 32
> >  mode = "http"
> >  bind = "*:80"
> > +httpchk = "GET /favicon.ico"
> Is this a sane default? So many services never implement a favicon.ico.
> 
> From the docs:
> 
> It defaults to " / " which is accessible by default on almost any server, but may be changed to any other URI
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>